### PR TITLE
Breol 31 news listing

### PR DESCRIPTION
**Pull request by xendk**
Adds news page

### CODE REVIEW

**Comment by bigslott** [comment]
Review ok

**Comment by xendk** [comment]
Excellent. You can grab this branch for BREOL-77, as we can't call the feature "done" without theming.

File: sites/all/themes/wille/template.php [hunk 1]
```diff
@@ -45,6 +45,20 @@ function wille_preprocess_wille_site_template(&$variables, $hook) {
...
 
 /**
+ * Implements THEME_preprocess_TEMPLATE();
+ *
+ * We preprocess and override the ctools content type here in the theme, because
+ * there are theming specific files and styles that shoudl be included.
+ */
+function wille_preprocess_breol_news_page(&$variables, $hook) {
+  $variables['organic_svg'] = file_get_contents(dirname(__FILE__) . "/svg/organic.svg");
+  if ($fid = variable_get('breol_news_page_image', NULL)) {
```
Comment (xendk) [inline sites/all/themes/wille/template.php]: Shouldn't be needed. $vars['image_file'] should already be the file object, added by breol_news_page_news_page().

File: sites/all/themes/wille/templates/panels/breol_news_page.tpl.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+/**
+ * @file
+ * Template for the news page.
+ *<div><?php print $image; ?></div>
+ */
+?>
+<div class="news-page">
+  <div class="news-page__cover-wrapper">
+    <div class="news-page__cover" style="background-image: url(<?php print $image_file_uri ?>);">
```
Comment (xendk) [inline sites/all/themes/wille/templates/panels/breol_news_page.tpl.php]: Background should also handle missing image.

**Pull request merged**